backend/tests/Feature/PublicationTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Publication;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class PublicationTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testPublicationsCanBeAssignedToAndQueriedForAUser()
+    {
+        $publication = Publication::factory()->create();
+        $user = User::factory()->create();
+        $user->publications()->attach($publication->id);
```

Comment:
So, what does this mean in this context?  Shouldn't a user's association to a publication always be through (or somehow related to, pseudocode-speak) a role?

Reply:
Yes, a user's association to a publication is relative to their role. Remember that users can have more than one role. Here's the breakdown:

- Not associated with publications at all: Application Admin
- Associated with publications directly: Publication Admin, Editor
- Associated with publications indirectly: Review Coordinator, Reviewer, Submitter

The publication-to-user association that's being performed here is a generic one. We'll need to integrate the "business logic" I outlined above later when we develop the code that handles role assignments.

Reply:
It's occurred to me that we should re-evaluate the need of having separate publications on an application instance, and if the complexity of implementing separate publications into the system is justified.

Reply:
Capturing from today's discussion:

We will continue implementing separate publications on an application instance, however, it appears that our permission library may not be designed in a way that satisfies our needs. That is, the permission library seems to be based on a 2-dimensional association (user/role) and we need a 3-dimensional association (user/role/model).

For now, we will reduce the scope of this issue to remove the direct linking between publications and users. We will continue to research solutions to address this association issue separately.